source/runner/Executor.ts

```diff
@@ -84,6 +84,7 @@ export class Executor {
         console.log("Found some message, writing it down")
       }
       const comment = githubResultsTemplate(results)
+      // TODO filter out Danger items that contain file/line info
```

Comment:
I believe this is the area where we could filter out results that contained a complete `{ file, line }` object and make inline comments. if `{ file, line }` is incomplete/missing, add them to the main Danger comment (I think this refers to your abstractions comment in https://github.com/danger/danger-js/issues/77#issuecomment-270195643, @orta).